gslib/tests/test_rpo.py

```diff
@@ -237,7 +237,16 @@ def test_s3_fails_for_set(self):
   @SkipForGS('Testing S3 only behavior.')
   def test_s3_fails_for_get(self):
     bucket_uri = self.CreateBucket()
-    stderr = self.RunGsUtil(['rpo', 'get', suri(bucket_uri)],
-                            return_stderr=True,
-                            expected_status=1)
-    self.assertIn('command can only be used for GCS buckets', stderr)
+    expected_status = 0 if self._use_gcloud_storage else 1
+    stdout, stderr = self.RunGsUtil(
+        ['rpo', 'get', suri(bucket_uri)],
+        return_stderr=True,
+        return_stdout=True,
+        expected_status=expected_status)
+    if self._use_gcloud_storage:
+      # TODO gcloud storage buckets list command doesn't raise an error when a
```

Comment:
I would say remove this TODO. The comment is slightly incorrect. gcloud storage shouldn't be raising an error for a list call as it is able to access the metadata of the biucket correctly. We are getting None here because of the format flag. The format flag is trying to access a field that is not present. In that case, returning None is the correct behavior. One solution can be that we populate the field's value as "Not available for S3" for the S3 Resource. You can create an internal bug. for this or add this to the bug (I can add it if you can send me the bug offline).

Reply:
Done!